resources/js/pack-it-forms.js

```diff
@@ -199,20 +199,17 @@ function parse_form_data_text(text) {
             return;  // Ignore line as we don't need anything from it.
         }
         var idx = 0;
-        if (field_name == "") {
+        if (field_name == null) {
             idx = index_of_field_name_sep(linenum, line, idx);
             field_name = line.substring(0, idx);
-            idx = index_of_field_value_start(linenum, line, idx) + 1;
+            idx = index_of_field_value_start(linenum, line, idx);
         }
-        var end_idx = line.indexOf("]", idx);
-        if (end_idx == -1) {
-            // Field continues on next line
-            field_value += line.substring(idx);
-        } else {
+        field_value += line.substring(idx);
+        idx = field_value.length - 1;
+        if (idx > 0 && field_value.charAt(idx) == "]" && field_value.charAt(idx - 1) != "`") {
```

Comment:
I think this assumes that the closing bracket of a multi-line field will always occur as the last character of the line. I don't think we would generate something that violates that constraint, but it is possible the user could insert stray whitespace accidentally when transmitting the message manually, etc. What do you think about instead matching on a regexp like:
```
(?<!`)]\S*$
```

Reply:
Good idea.